Gateway/Http/Client/TransactionCapture.php

```diff
@@ -76,9 +76,9 @@ public function placeRequest(TransferInterface $transferObject): array
         $request = $transferObject->getBody();
         $headers = $transferObject->getHeaders();
 
-        $service = new Modification(
-            $this->adyenHelper->initializeAdyenClient($transferObject->getClientConfig()['storeId'])
-        );
+        $client = $this->adyenHelper->initializeAdyenClient();
+        $service = $this->adyenHelper->createAdyenCheckoutService($client);
```

Comment:
I've introduced a PR to get rid of that decoupling

https://github.com/Adyen/adyen-magento2/pull/2193